condition.go

```diff
@@ -117,3 +117,37 @@ func Lte(column string, value interface{}) Builder {
 		return buildCmp(d, buf, "<=", column, value)
 	})
 }
+
+// Lk is `LIKE`.
+// When value is nil, do nothing.
+// When value is a slice, do nothing.
+// Otherwise it will be translated to `LIKE`.
+func Lk(column string, value interface{}) Builder {
+	return BuildFunc(func(d Dialect, buf Buffer) error {
+		if value == nil {
```

Comment:
IMO: it would be better avoid some logic here and pass arguments as is to underlayed level. if aguments is not valid, user will be notified about this via query builder error or via sql error.